mtg_card_identifier/CardScannerFile.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  CardScannerFile.swift
+//  mtg_card_identifier
+//
+//  Created by mac on 8/16/23.
+//
+
+import SwiftUI
+import UIKit
+import VisionKit
+
+struct CardScanner: UIViewControllerRepresentable {
+    
+    //    @Binding var startScanning: Bool
+    @Binding var scanedCard: [String]
```

Comment:
this should probably be more generic, like `scannedText`.

Reply:
We also might need more than just the Strings here... we can leave it for now, but I'm our logic for creating a Card from an array of Strings might get a bit complicated and the location of the String might eventually be needed.